boards/microbit_v2/README.md

```diff
@@ -17,7 +17,83 @@ following sensors:
 
 First, follow the [Tock Getting Started guide](../../../doc/Getting_Started.md)
 
-## Flashing the kernel
+## Bootloader
+
+Tock uses [Tock Bootloader](https://github.com/tock/tock-bootloader) to program devices.
+
+As MicroBit v2 has an on board debugger that provides several ways of programming it, is shipped without an actual bootloader.
+
+There are two ways for flashing the bootloader:
+ 1. Using the MicroBit USB drive
+ 2. Using openocd
+
+### Using the MicroBit USB Drive
+
+Connect then MicroBit to the computer. A USB drive labeled `MICROBIT` should show up. 
+
+Drag and drop the `bootloader/tock-bootloader.microbit_v2.v1.1.0.bin` to the `MICROBIT` drive and wait for a few seconds.
+
+The board will reset and the bootloader should be running on it. To check whether it's working, press and hold the Button A while pressing the reset button. The Microphone LED should light up.
+
+### Using openocd 
+Use the `make flash-bootloader` command to flash [Tock Bootloader](https://github.com/tock/tock-bootloader) to the board.
+
+```bash
+$ make flash-bootloader
```

Comment:
Can you also describe how to build the bootloader yourself?

As well as that I'm not sure if we want binary blobs in Tock.

Reply:
> Can you also describe how to build the bootloader yourself?
> 
It never goes away, the USB drive will always appear, it is provided by the debugger not the nrf52 chip.

> As well as that I'm not sure if we want binary blobs in Tock.

I just duplicated Brad's Nano 33 approach :)

Reply:
Done.